.github/workflows/publish.yml

```diff
@@ -52,10 +59,15 @@ jobs:
         working-directory: base
 
       - name: Base Images > Docker Build & Push
-        if: ${{ github.event_name == 'push' && github.ref == 'refs/heads/master' }}
+        if: ${{ github.event_name == 'push' && github.ref == 'refs/heads/master' && steps.changes.outputs.base != 'true'  }}
         run: ./docker_tags.sh -p
         working-directory: base
 
+      - name: Base Images > Docker Build & Push
+        if: ${{ github.event_name == 'push' && github.ref == 'refs/heads/master' && steps.changes.outputs.base == 'true' }}
```

Comment:
I think not everyone knows about dorny/paths-filter ; can we add a comment here? Or improve the `name` of this step, to inform readers of what is going on?

Reply:
Added comment